plugins/sigma.renderers.linkurious/canvas/sigma.canvas.nodes.def.js

```diff
@@ -194,6 +206,10 @@
       context.arc(x, y, size, 0, Math.PI * 2, true);
       context.closePath();
       context.fill();
+
+      if (borderSize > 0) {
```

Comment:
ok.  I'll make the change and resubmit the pull request when I get your ok for the `borderColor` above